src/main/java/com/redislabs/modules/rejson/JReJSON.java

```diff
@@ -80,7 +80,7 @@
         }
     }
 
-	private Pool<Jedis> client;
+	protected Pool<Jedis> client;
```

Comment:
Revealing an internal object increases the chance of it being abused. Keep this one private.

```suggestion
	private Pool<Jedis> client;
```

Reply:
What do you think of adding an accessor for the underlying Jedis Pooled client? I'll add that to a patch commit.

Reply:
Skip testing this.

Reply:
Addressed